content/introduction/what-is-dolt.md

```diff
@@ -0,0 +1,157 @@
+| US                    | arin        | 939513600       | United States       | 3639955711 | 3639955456 | USA                   | bqtrbbrsgie7u4fsgph0t12j94ofefml | 2020-12-11 01:30:36.075 +0000 UTC | NULL                    | NULL          | NULL              | NULL         | NULL      | NULL        | NULL                    | u8pnf1o0mus2d8mok0083t5lpj190j5f | 2020-12-10 01:30:29.088 +0000 UTC | added     |
+| US                    | arin        | 1607385600      | United States       | 3639955967 | 3639955712 | USA                   | bqtrbbrsgie7u4fsgph0t12j94ofefml | 2020-12-11 01:30:36.075 +0000 UTC | NULL                    | NULL          | NULL              | NULL         | NULL      | NULL        | NULL                    | u8pnf1o0mus2d8mok0083t5lpj190j5f | 2020-12-10 01:30:29.088 +0000 UTC | added     |
+| US                    | arin        | 939513600       | United States       | 3639956479 | 3639955968 | USA                   | bqtrbbrsgie7u4fsgph0t12j94ofefml | 2020-12-11 01:30:36.075 +0000 UTC | NULL                    | NULL          | NULL              | NULL         | NULL      | NULL        | NULL                    | u8pnf1o0mus2d8mok0083t5lpj190j5f | 2020-12-10 01:30:29.088 +0000 UTC | added     |
+| US                    | arin        | 1607385600      | United States       | 3639957503 | 3639956480 | USA                   | bqtrbbrsgie7u4fsgph0t12j94ofefml | 2020-12-11 01:30:36.075 +0000 UTC | NULL                    | NULL          | NULL              | NULL         | NULL      | NULL        | NULL                    | u8pnf1o0mus2d8mok0083t5lpj190j5f | 2020-12-10 01:30:29.088 +0000 UTC | added     |
+| US                    | arin        | 939513600       | United States       | 3639959551 | 3639957504 | USA                   | bqtrbbrsgie7u4fsgph0t12j94ofefml | 2020-12-11 01:30:36.075 +0000 UTC | NULL                    | NULL          | NULL              | NULL         | NULL      | NULL        | NULL                    | u8pnf1o0mus2d8mok0083t5lpj190j5f | 2020-12-10 01:30:29.088 +0000 UTC | added     |
++-----------------------+-------------+-----------------+---------------------+------------+------------+-----------------------+----------------------------------+-----------------------------------+-------------------------+---------------+-------------------+--------------+-----------+-------------+-------------------------+----------------------------------+-----------------------------------+-----------+
+```
+
+
+### Rollback
+Continuing with this example, let's suppose this is causing a production issue, and we need to rollback urgently. Dolt makes that easy:
+```
+mysql> INSERT INTO dolt_branches (name, hash) VALUES  ('production' , 'u8pnf1o0mus2d8mok0083t5lpj190j5f');
```

Comment:
Maybe use the new dolt_checkout function here. Seems more user friendly than an insert into the branches table.

Reply:
But they are doing different things, right? In Git this would be a `reset`:
```
NAME
       git-reset - Reset current HEAD to the specified state

SYNOPSIS
       git reset [-q] [<tree-ish>] [--] <paths>...
       git reset (--patch | -p) [<tree-ish>] [--] [<paths>...]
       git reset [--soft | --mixed [-N] | --hard | --merge | --keep] [-q] [<commit>]
```

We don't support these semantics.

Reply:
We have GIT_RESET. That was our first one.

Reply:
Right, but `dolt reset` does not do the same thing, it just nukes the working set. It does not allow you to point the branch at a different revision.